salt/modules/boto_route53.py

```diff
@@ -106,6 +106,10 @@ def _get_split_zone(zone, _conn, private_zone):
     return False
 
 
+def _is_retryable_error(exception):
+    return exception.code not in ["SignatureDoesNotMatch"]
```

Comment:
Just a question: is there a reason to look up a single-item list? Maybe just use `!=`?

Reply:
Good question. The original code author is @kaidokert, I just ported this to master.

Reply:
I have changed this to `!=` and am working on unit tests.